airlock/business_logic.py

```diff
@@ -539,20 +574,35 @@ def from_dict(cls, attrs) -> Self:
             },
         )
 
-    def approved_for_release(self):
-        """
-        A file is approved for release if it has been approved by two reviewers
+    def status(self) -> RequestFileReviewStatus:
```

Comment:
Would it be worth calling this something like `get_status`, so it's clear that its a calculated status based on other things? We have a lot of `status`es that are attributes set on something. I mainly noted it because there's a change on line 603 that includes `.status` and then one on line 770 that includes `.status()` and I had to come back here to remind myself which sort of status this was.
Maybe the fact that you have to call it with parentheses is enough

Reply:
That's a fair point, will do.